glean-core/src/metrics/ping.rs

```diff
@@ -0,0 +1,28 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::error::Result;
+use crate::Glean;
+
+#[derive(Debug, Clone)]
+/// Stores information about a ping.
```

Comment:
Can we explain why this is needed, e.g. to make it possible to send pings with partial information, for example, no client id.